zha/application/platforms/sensor/__init__.py

```diff
@@ -1947,3 +1949,14 @@ class DanfossMotorStepCounter(Sensor):
     _attribute_name = "motor_step_counter"
     _attr_translation_key: str = "motor_stepcount"
     _attr_entity_category = EntityCategory.DIAGNOSTIC
+
+
+@MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_ELECTRICAL_CONDUCTIVITY)
+class ElectricalConductivity(Sensor):
+    """Electrical Conductivity sensor."""
+
+    _attribute_name = "measured_value"
+    _attr_device_class: SensorDeviceClass = SensorDeviceClass.CONDUCTIVITY
+    _attr_state_class: SensorStateClass = SensorStateClass.MEASUREMENT
+    _attr_translation_key: str = "electrical_conductivity"
```

Comment:
I'm not sure we even need a translation key for this, since "Conductivity" is already used as a translated name due to the `SensorDeviceClass.CONDUCTIVITY` if a specific translation key isn't defined.
(Coming from here: https://github.com/home-assistant/core/blob/b2377d6da35945f34e0ad33fc3481479c3c35b51/homeassistant/components/sensor/strings.json#L172-L174)

If we want to introduce a new translation key for this, we need to make sure to add it to the ZHA integration's `strings.json` in Home Assistant Core when merging a lib bump with this PR included. The English string should likely be "Electrical conductivity" in that case(?)

Reply:
_Conductivity_ sounds good to me.